data.sql

```diff
@@ -1,32 +1,442 @@
+('Plantmon', 'Nov 15 2021',2, true, 5.7),
+('Squirtle', 'Apr 02 1993',3, false, 12.13),
+('Angemon', 'Jun 12 2005',1, true, 45), 
+('Boarmon', 'Jun 07 2005',7, true, 20.4),
+('Blossom', 'Oct 13 1998',3, true, 17),
+('Ditto', 'May 14 2022',4, true, 22);
 
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 5, 'Charmander', '2020-02-08', 0, 'false', '-11', 'Unspecified' );
 
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 6, 'Plantmon', '2021-11-15', 2, 'true', '-5.7', 'Unspecified' );
+-- insert data into owners table
+INSERT INTO  owners(full_name, age) VALUES('Sam Smith ', 34), ('Jennifer Orwell', 19 ),
+('Bob', 45 ),('Melody Pond', 77 ),('Dean Winchester', 14 ),('Jodie Whittaker', 38 );
+
+-- insert data for species
+
+INSERT INTO  species(name) VALUES('Pokemon'), ('Digimon');
+
+-- Modify your inserted animals so it includes the species_id value
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') 
+
+WHERE name LIKE '%mon';
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') 
+WHERE name NOT LIKE '%mon';
+
+-- Modify your inserted animals to include owner information (owner_id)
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Sam Smith'
+    )
+WHERE name = 'Agumon';
+UPDATE 1
+
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Jennifer Orwell'
+    )
+WHERE name = 'Gabumon';
+UPDATE 1
+
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Bob'
+    )
+WHERE name = 'Plantmon' AND 'Devimon';
+
+
+ UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Bob'
+    )
+                              ^
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Bob'
+    )
+WHERE name = 'Plantmon' OR name= 'Devimon';
+
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Melody Pond'
+    )
+WHERE name = 'Charmander' OR name= 'Squirtle' OR name = 'Blossom';
+
+ UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Dean Winchester'
+    )
+WHERE name = 'Angemon' OR name= 'Boarmon';
+
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Jennifer Orwell'
+    )
+WHERE name = 'Pikachu';
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Sam Smith '
+    )
+WHERE name = 'Agumon';
+SELECT * FROM animals;
+
+-- insert data into vets table
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Vet William Tatcher', 45, 'Apr 23 2000'),
+('Vet Maisy Smith', 26 ,'Jan 17 2019'),
+('Vet Stephanie Mendez', 64, 'May 04 1981'),
+('Vet Jack Harkness', 38,'Jun 08 2008');
+
```

Comment:
- [ ] You were instructed to add the following data for `vets` table 👇🏻  but the name is different, 

  - Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
  - Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
  - Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
  - Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
- [ ] Kindly remove the `Vet` in front of your names, your code should be like this:

```sql
   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('William Tatcher', 45, '2000-04-23');

   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Maisy Smith', 26, '2019-01-17');

   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Stephanie Mendez', 64, '1981-05-04');

   INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Jack Harknes', 38, '2008-06-08');
```